src/core/MOM.F90

```diff
@@ -2856,6 +2880,12 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, restart_CSp, &
             CS%ntrunc, cont_stencil=CS%cont_stencil)
   endif
 
+  !Set OBC segment data update period
+  if (associated(CS%OBC) .and. CS%dt_obc_seg_period > 0.0) then
+    CS%dt_obc_seg_interval = real_to_time(CS%dt_obc_seg_period)
+    CS%dt_obc_seg_time = Time + CS%dt_obc_seg_interval
```

Comment:
I need to test this, but I think setting this to `Time + CS%dt_obc_seg_interval` results in the tracer reservoirs not being initialized at the first time step and only getting initialized at the first obc segment update period. In both NWA and NEP regional domains, water inflowing from the boundary is very low in alkalinity, oxygen, and other tracers when the model first starts running, as if the reservoirs were initialized at 0.

Reply:
I guess the reservoirs should also be initialized by `fill_obgc_segments`. Need to look into this more.